locations/spiders/travelodge_gb.py

```diff
@@ -0,0 +1,17 @@
+import html
+
+from scrapy.spiders import SitemapSpider
+
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class TravelodgeGBSpider(SitemapSpider, StructuredDataSpider):
+    name = "travelodge_gb"
+    item_attributes = {"brand": "Travelodge", "brand_wikidata": "Q9361374"}
```

Comment:
Did we have this previously, then remove for some reason?

https://github.com/search?q=repo%3Aalltheplaces%2Falltheplaces+Q9361374&type=code